app/uk/gov/hmrc/minorentityidentificationfrontend/api/controllers/JourneyController.scala

```diff
@@ -61,23 +62,24 @@ class JourneyController @Inject()(val authConnector: AuthConnector,
       authorised().retrieve(internalId) {
         case Some(authInternalId) =>
           journeyService.createJourney(req.body, authInternalId).map(
-            journeyId =>
+            journeyId => {
               businessEntity match {
                 case OverseasCompany => Created(Json.obj(
-                  "journeyStartUrl" -> s"${appConfig.selfUrl}${controllerRoutes.CaptureUtrController.show(journeyId).url}"
+                  journeyStartUrl -> s"${appConfig.selfUrl}${controllerRoutes.CaptureUtrController.show(journeyId).url}"
                 ))
                 case Trusts =>
                   auditService.auditJourney(journeyId, authInternalId)
                   Created(Json.obj(
-                    "journeyStartUrl" -> (req.body.continueUrl + s"?journeyId=$journeyId")
+                    journeyStartUrl -> s"${appConfig.selfUrl}${trustControllerRoutes.TrustUtrController.show(journeyId).url}"
```

Comment:
Only redirect if the new feature switch is set here. If false direct to the continue url as was there previously